draft-ietf-wimse-arch.md

```diff
@@ -75,6 +75,10 @@ Identity proxy is an intermediary that can inspect, replace or augment workload
 
 Attestation is the function through which a task verifies the identity of a separate Workload. (TBD: sync definition with reference RaTS architecture)
 
+* Workload Identity Token
```

Comment:
Maybe call it WIMSE token?

Reply:
This is what it is called in the S2S spec,  we would need to change the name there.